osf/models/user.py

```diff
@@ -672,6 +674,16 @@ def is_institutional_curator(self, node):
             is_curator=True,
         ).exists()
 
+    def has_institutional_request(self, node):
+        """
+        Checks if user a has requested a node using the institutional access request feature.
+        """
+        return NodeRequest.objects.filter(
+            request_type=NodeRequestTypes.INSTITUTIONAL_REQUEST.value,
+            target=node,
+            creator=self,
+        ).exists()
```

Comment:
Just to be sure, is there any place we can cache this info, such as a request object, rather than relying on this query, the same way we did with caching on the contributor object if it's a curator? Same reason as before, to avoid making this query for every contributor that gets serialized.